consensus/consensus-types/src/common.rs

```diff
@@ -418,22 +418,16 @@ impl Payload {
                 Payload::QuorumStoreInlineHybrid(b2, p3, m3)
             },
             (
-                Payload::QuorumStoreInlineHybrid(inline_batches, proofs, limit),
-                Payload::OptQuorumStore(opt_qs),
+                Payload::QuorumStoreInlineHybrid(_inline_batches, _proofs, _limit),
+                Payload::OptQuorumStore(_opt_qs),
             )
             | (
-                Payload::OptQuorumStore(opt_qs),
-                Payload::QuorumStoreInlineHybrid(inline_batches, proofs, limit),
+                Payload::OptQuorumStore(_opt_qs),
+                Payload::QuorumStoreInlineHybrid(_inline_batches, _proofs, _limit),
             ) => {
-                let execution_limits = PayloadExecutionLimit::max_txns_to_execute(limit);
-                let converted_payload = OptQuorumStorePayload::new(
-                    inline_batches.into(),
-                    Vec::new().into(),
-                    proofs.into(),
-                    execution_limits,
-                );
-                let opt_qs3 = opt_qs.extend(converted_payload);
-                Payload::OptQuorumStore(opt_qs3)
+                unimplemented!(
```

Comment:
hmm, where do we use it? only in DAG? otherwise one proposal can have hybrid while another has opt qs?

Reply:
Yes, it's only needed for DAG. we can revisit later for DAG. I might just need to join futures in that case.